### PR TITLE
fix(ranger): define python interpreter for ranger-usersync setup.py

### LINKED ISSUE #405: ranger: hardcoded Python path causing issues on RHEL/CentOS 8
The python command is hardcoded in setup.py for ranger-usersync and ranger-tagsync component.

- https://github.com/apache/ranger/blob/master/unixauthservice/scripts/setup.py

- https://github.com/apache/ranger/blob/master/tagsync/scripts/setup.py

By the way, the patch RANGER-2546[https://issues.apache.org/jira/browse/RANGER-2546] which resolves the same issue for ranger-admin and ranger-kms has been backported in ranger project.

### PR DESCRIPTION
**Pull request by nschung**
#### Which issue(s) this PR fixes:

Fixes #405 

#### Additional comments:

To make clear that you license your contribution under
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0) and that you give permission to [TOSIT](https://www.tosit.io/),
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

File: tdp_vars_defaults/ranger/ranger.yml [hunk 1]
```diff
@@ -166,3 +166,6 @@ jmx_exporter:
   password: "{{ jmxremote_password }}"
   lowercaseOutputName: false
   lowercaseOutputLabelNames: false
+
+#python interpreter
```
Comment (rpignolet) [inline tdp_vars_defaults/ranger/ranger.yml:170]: No need to put comment which duplicate the variable name.

File: roles/ranger/usersync/tasks/install.yml [hunk 2]
```diff
@@ -27,6 +27,12 @@
     dest: "{{ ranger_usersync_install_dir }}"
     state: link
 
+- name: Define python interpreter
```
Comment (rpignolet) [inline roles/ranger/usersync/tasks/install.yml:30]: Can you change with something like this : `Patch python interpreter for Ranger Usersync setup.py` which is better for me.

File: tdp_vars_defaults/ranger/ranger.yml (hunk 1 above)
Comment (nschung) [reply tdp_vars_defaults/ranger/ranger.yml]: Done

File: roles/ranger/usersync/tasks/install.yml (hunk 2 above)
Comment (nschung) [reply roles/ranger/usersync/tasks/install.yml]: Done

**Review by rpignolet — APPROVED** [review]

**Review by gboutry — APPROVED** [review]

**Pull request merged**